src/test/java/tests/auth/InstitutionAuthServiceTest.java

```diff
@@ -80,4 +80,19 @@ public void adminInvalidateToken() {
                 .body("isSuccess", equalTo(true));
     }
 
+    @Test()
```

Comment:
Eğer anotasyonun içerisine bir şey yazmayacaksak parantezleri eklememek her zaman en doğru yöntemdir
```suggestion
    @Test
```